src/test/searchAddressFromZipcode.test.ts

```diff
@@ -0,0 +1,105 @@
+import { expect } from '@jest/globals';
+import { addresses } from './constants/masterData/address';
+import { expectedadAddressDataListIncludHyphen } from './constants/masterData/address';
+import { expectedadAddressDataListExcludingHyphen } from './constants/masterData/address';
```

Comment:
@marumoto-goq 
まとめてかけます。
```suggestion
import { expectedadAddressDataListIncludHyphen, expectedadAddressDataListExcludingHyphen } from './constants/masterData/address';
```